src/cata_tiles.cpp

```diff
@@ -1251,8 +1293,8 @@ bool cata_tiles::draw_from_id_string(std::string id, TILE_CATEGORY category,
         if (std::find(begin(display_subtiles), end, multitile_keys[subtile]) != end) {
             // append subtile name to tile and re-find display_tile
             return draw_from_id_string(
-                       std::move(id.append("_", 1).append(multitile_keys[subtile])), x, y, -1, rota, ll,
-                       apply_night_vision_goggles);
+                std::move( id.append( "_", 1 ).append( multitile_keys[subtile] ) ), x, y,
```

Comment:
That move here (in this usage) is necessary. Its parameter `id.append().append()` is a temporary object and will be moved into the parameter of the called function anyway.